### PR TITLE
[TASK] Add composer.json and template

### PR DESCRIPTION
**Pull request by kitsunet**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        "ext-SPL": "*",
+        "ext-json": ">=1.2.0",
+        "ext-mbstring": "*",
+        "ext-reflection": "*",
+        "doctrine/orm": "~2.4.0",
+        "doctrine/migrations": "~1.0.0",
+        "symfony/yaml": "~2.5.0",
+        "symfony/dom-crawler": "~2.5.0",
+        "symfony/console": "~2.0",
+        "composer/installers": "~1.0.2"
```
Comment (aertmann) [inline composer.json]: should this be replaced with "neos/composer-plugin": "*" as in master?

related to https://github.com/neos/flow-development-collection/pull/15

**Comment by aertmann** [comment]
Do you plan to synchronize this one with the master one? both have something the other doesn't..

File: composer.json (hunk 1 above)
Comment (kitsunet) [inline composer.json]: Good point, that was a separate change. Ok let me go though that and push everything needed to make this work...

**Comment by kitsunet** [comment]
re composer plugin and installers you mean? Yes that should be synchronised, I'll take care.
This commit was a test of how this could go automatically. I basically didn't touch the composer.json manually.

**Comment by aertmann** [comment]
:+1:

**Comment by dfeyer** [comment]
Yesterday I install the neos dev collection, but the composer.json does not require neos/composer-plugin ... maybe i'm wrong but should we add this in the require section ?

**Comment by kitsunet** [comment]
no, neos shouldn't require it. It's fine if only flow and the build essentials do. pushing a new chnage in a few minutes...

**Comment by kitsunet** [comment]
There we go, lets merge this... Hard to test anyway

**Comment by dfeyer** [comment]
:+1: by reading

**Pull request merged**